packages/react-server/src/ReactFizzServer.js

```diff
@@ -3870,11 +3869,24 @@ function abortTask(task: Task, request: Request, error: mixed): void {
       ) {
         const postponeInstance: Postpone = (error: any);
         logPostpone(request, postponeInstance.message, errorInfo, null);
+        if (request.trackedPostpones !== null && segment !== null) {
+          trackPostpone(request, request.trackedPostpones, task, segment);
+          finishedTask(request, task.blockedBoundary, segment);
+
+          // If this boundary was still pending then we haven't already cancelled its fallbacks.
+          // We'll need to abort the fallbacks, which will also error that parent boundary.
+          boundary.fallbackAbortableTasks.forEach(fallbackTask =>
```

Comment:
This is the unfortunate case where we hadn't finished the fallback yet which should only be the case if it suspends or has too many awaits.